Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -19210,7 +19383,9 @@
 		};
 		5C400E6A29DB8AB500446F6F /* PBXTargetDependency */ = {
 			isa = PBXTargetDependency;
-			platformFilter = ios;
+			platformFilters = (
+				ios,
+			);
```

Comment:
Just confirming that this should be safe to commit. AFAICT, the `platformFilters` key has been recognized since Xcode 13, which is as far back as our support goes.